tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -22,16 +22,27 @@ pr:
       - tools/apiview/parsers/cpp-api-parser
 
 variables:
-  ArtifactName: 'apiview'
-  VcpkgRelease: '2024.08.23'
+  - template: /eng/pipelines/templates/variables/image.yml
+  - template: /eng/pipelines/templates/variables/globals.yml
+  - name: ArtifactName
```

Comment:
It doesn't look like ArtifactName or VcpkgRelease are used anywhere and if they aren't please remove.